OpenAI-DotNet/Realtime/RealtimeEndpointWebRTC.cs

```diff
@@ -0,0 +1,203 @@
+﻿// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using OpenAI.Extensions;
+using Org.BouncyCastle.Ocsp;
+using SIPSorcery.Media;
+using SIPSorcery.Net;
+using SIPSorceryMedia.Abstractions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace OpenAI.Realtime
+{
+    public sealed class RealtimeEndpointWebRTC : OpenAIBaseEndpoint
+    {
+        private const string OPENAI_DATACHANNEL_NAME = "oai-events";
+
+        public readonly AudioEncoder AudioEncoder;
+
+        public readonly AudioFormat AudioFormat;
+
+        internal RealtimeEndpointWebRTC(OpenAIClient client) : base(client)
+        {
+            AudioEncoder = new AudioEncoder(includeOpus: true);
+            AudioFormat = AudioEncoder.SupportedFormats.Single(x => x.FormatName == AudioCodecsEnum.OPUS.ToString());
+        }
+
+        protected override string Root => "realtime";
+
+        protected override bool? IsWebSocketEndpoint => false;
+
+        private RTCPeerConnection rtcPeerConnection;
+        public RTCPeerConnection RTCPeerConnection => rtcPeerConnection;

```

Comment:
Should be fine as auto propery
```suggestion
        public RTCPeerConnection RTCPeerConnection { get; private set; }

```

Reply:
Actually that didn't end up being needed. Will remove.